plugin/mutate/source/dextool/plugin/mutate/backend/database/standalone.d

```diff
@@ -411,6 +440,17 @@ struct Database {
             rval = MutationStatusId(res.peek!long(0));
         return rval;
     }
+    // Returns: the status of the mutant
+    Nullable!Mutation.Status getMutationStatus(const MutationId id) @trusted {
```

Comment:
After thinking about this function and the two other four loc and kind I think they can be removed because all that info is already accessible via getMutation in one call

Reply:
getMutation gives a MutationEntry, which does not contain status or kind. Therefore I created the functions for it, so that it would be easier to access specific items.